src/FlapperUniV2.sol

```diff
@@ -164,12 +159,9 @@ contract FlapperUniV2 {
         amtOut = _amtInFee * reserveOut / (reserveIn * 1000 + _amtInFee);
     }
 
-    function kick(uint256 lot, uint256) external auth returns (uint256) {
+    function exec(uint256 lot) external auth returns (uint256) {
```

Comment:
I guess if we are removing that unneeded param we can also remove the return value.

Reply:
Removed.